quiz.rb

```diff
@@ -14,24 +14,32 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  paragraph.split(" ")
```

Comment:
make sure the argument in the method declaration and the argument you manipulate inside of the method match. in these methods you declare `words`, but manipulate `paragraph`